profiles/build.profile.js

```diff
@@ -22,7 +22,6 @@ var profile = {
                 'dojox/gfx/svgext',
                 'dojox/gfx/shape'
             ],
-            targetStylesheet: 'app/resources/App.css',
```

Comment:
I ran another build just to be sure that it's working. No errors and `App.css` is the only CSS file that's loaded.

I'm pretty sure that `targetStylesheet` [is a xstyle thing](https://github.com/kriszyp/xstyle#building-with-amd-plugin).